lib/internal/test_runner/tap_stream.js

```diff
@@ -0,0 +1,187 @@
+'use strict';
+const {
+  ArrayPrototypeForEach,
+  ArrayPrototypeJoin,
+  ArrayPrototypePush,
+  ArrayPrototypeShift,
+  ObjectEntries,
+  StringPrototypeReplace,
+  StringPrototypeSplit,
+} = primordials;
+const { isError } = require('internal/util');
+const { Readable } = require('stream');
+const { inspect } = require('util');
+const kFrameStartRegExp = /^ {4}at /;
+const kLineBreakRegExp = /\n|\r\n/;
+const inspectOptions = { colors: false, breakLength: Infinity };
+let testModule; // Lazy loaded due to circular dependency.
+
+function lazyLoadTest() {
+  if (testModule === undefined) {
+    testModule = require('internal/test_runner/test');
+  }
+
+  return testModule;
+}
+
+class TapStream extends Readable {
+  #buffer;
+  #canPush;
```

Comment:
Not blocking, especially since this isn't a production hot path but private properties are still significantly slower than non-exported symbol properties.

Reply:
@jasnell not on lasted merged v8 : https://github.com/nodejs/node/issues/42229#issuecomment-1063314311

Reply:
Marking as resolved based on the previous comment.